app/src/main/java/org/schabi/newpipe/util/Localization.java

```diff
@@ -98,23 +90,24 @@ public static ContentCountry getPreferredContentCountry(final Context context) {
         return new ContentCountry(contentCountry);
     }
 
-    public static Locale getPreferredLocale(final Context context) {
+    private static Locale getLocaleFromPrefs(final Context context, @StringRes final int prefKey) {
```

Comment:
Could you move this method below the `assureCorrectAppLanguage` one, in order to have one block of private methods and one of public ones?

Reply:
Sure.